charts/newrelic-infrastructure-beta/templates/agent-configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.config }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic.labels" . | nindent 4 }}
+  name: {{ template "newrelic.fullname" . }}-agent
+data:
+    newrelic-infra.yml: |
```

Comment:
This should have 2 spaces. The `indent` below 4. We should add a YAML linter to ther charts and the templated exist from the `ci` values...